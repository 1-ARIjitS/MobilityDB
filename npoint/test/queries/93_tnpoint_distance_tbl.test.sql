-------------------------------------------------------------------------------

SELECT round(MAX(maxValue(ST_SetSRID(t1.g, 5676) <-> t2.temp))::numeric, 6) FROM tbl_geom_point t1, tbl_tnpoint t2;
SELECT round(MAX(maxValue(t1.np <-> t2.temp))::numeric, 6) FROM tbl_npoint t1, tbl_tnpoint t2;
SELECT round(MAX(maxValue(t1.temp <-> ST_SetSRID(t2.g, 5676)))::numeric, 6) FROM tbl_tnpoint t1, tbl_geom_point t2;
SELECT round(MAX(maxValue(t1.temp <-> t2.np))::numeric, 6) FROM tbl_tnpoint t1, tbl_npoint t2;
SELECT count(*) FROM tbl_tnpoint t1, tbl_tnpoint t2 WHERE t1.temp <-> t2.temp IS NOT NULL;

-------------------------------------------------------------------------------
