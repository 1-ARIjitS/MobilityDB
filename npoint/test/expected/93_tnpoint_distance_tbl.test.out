<<<<<<< HEAD
<<<<<<< HEAD
=======
>>>>>>> Improve temporal cache
SELECT round(MAX(maxValue(ST_SetSRID(t1.g, 5676) <-> t2.temp))::numeric, 6) FROM tbl_geom_point t1, tbl_tnpoint t2;
   round    
------------
 125.449444
<<<<<<< HEAD
(1 row)

SELECT round(MAX(maxValue(t1.np <-> t2.temp))::numeric, 6) FROM tbl_npoint t1, tbl_tnpoint t2;
   round    
------------
 121.839892
(1 row)

SELECT round(MAX(maxValue(t1.temp <-> ST_SetSRID(t2.g, 5676)))::numeric, 6) FROM tbl_tnpoint t1, tbl_geom_point t2;
   round    
------------
 125.449444
(1 row)

SELECT round(MAX(maxValue(t1.temp <-> t2.np))::numeric, 6) FROM tbl_tnpoint t1, tbl_npoint t2;
   round    
------------
 121.839892
=======
SELECT MAX(maxValue(ST_SetSRID(t1.g, 5676) <-> t2.temp)) FROM tbl_geompoint t1, tbl_tnpoint t2;
       max        
------------------
 125.449443600402
=======
>>>>>>> Improve temporal cache
(1 row)

SELECT round(MAX(maxValue(t1.np <-> t2.temp))::numeric, 6) FROM tbl_npoint t1, tbl_tnpoint t2;
   round    
------------
 121.839892
(1 row)

SELECT round(MAX(maxValue(t1.temp <-> ST_SetSRID(t2.g, 5676)))::numeric, 6) FROM tbl_tnpoint t1, tbl_geom_point t2;
   round    
------------
 125.449444
(1 row)

<<<<<<< HEAD
SELECT MAX(maxValue(t1.temp <-> t2.np)) FROM tbl_tnpoint t1, tbl_npoint t2;
       max        
------------------
 121.839892066468
>>>>>>> Synchronize npoint with latest MobilityDB version
=======
SELECT round(MAX(maxValue(t1.temp <-> t2.np))::numeric, 6) FROM tbl_tnpoint t1, tbl_npoint t2;
   round    
------------
 121.839892
>>>>>>> Improve temporal cache
(1 row)

SELECT count(*) FROM tbl_tnpoint t1, tbl_tnpoint t2 WHERE t1.temp <-> t2.temp IS NOT NULL;
 count 
-------
   100
(1 row)

